### PR TITLE
[SPARK-8771][TRIVIAL] Add a version to the deprecated annotation for the actorSystem

### PR DESCRIPTION
**Pull request by holdenk**

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/SparkEnv.scala [hunk 1]
```diff
@@ -76,7 +76,7 @@ class SparkEnv (
     val conf: SparkConf) extends Logging {
 
   // TODO Remove actorSystem
-  @deprecated("Actor system is no longer supported as of 1.4")
+  @deprecated("Actor system is no longer supported as of 1.4", "1.4")
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/SparkEnv.scala]: Yeah, I double-checked and that's the only one. Good to fix. Nit: write 1.4.0? looks like the other messages do it that way.

Comment (holdenk) [inline core/src/main/scala/org/apache/spark/SparkEnv.scala]: Will do.

**Comment by sarutak** [comment]
retest this please.

**Comment by andrewor14** [comment]
Merging into master 1.4. Thanks @holdenk.

**Pull request closed without merging**